app/views/registrations/_form.haml

```diff
@@ -57,4 +57,4 @@
     %p.terms.text-center#terms
       = t('registrations.new.terms', terms_link: link_to(t('registrations.new.terms_link'), terms_path, target: "_blank")).html_safe
 
-  = f.submit t('registrations.new.sign_up'), class: "btn btn-block btn-large btn-default", data: {disable_with: t('registrations.new.submitting')}
+  = f.submit t('registrations.new.sign_up'), class: "btn btn-block btn-large btn-primary", data: {disable_with: t('registrations.new.submitting')}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.